### PR TITLE
CDA data model starting point

### PR DESCRIPTION
**Pull request by kreinold**
Initial check in for [CDA-22](https://broadworkbench.atlassian.net/browse/CDA-22).  

- ccdh/MVPv0/entities.json is a copy taken from [CCDH MVPv0](https://github.com/cancerDHC/ccdhmodel/blob/mvpv0/json-schema/entities.json) to record our starting point
- files under src/schema/json split out the different classes to make them more manageable as we embark on customizing the data model to meet CDA MVP requirements.

### CODE REVIEW

File: src/schema/json/datatypes.json [hunk 1]
```diff
@@ -0,0 +1,98 @@
+// Extracted from CCDH MVPv0
+// Defining complex data types
+{
+    "$id": "https://example.org/cda-data-model/datatypes#",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "definitions": {
+        "CodeableConcept": {
```
Comment (pshapiro4broad) [inline src/schema/json/datatypes.json:7]: This appears to duplicate the definition in `entities.json`. Do both fields need to define the same things? If so, how do we make sure that the file contents stay synchronized?

**Review by pshapiro4broad — APPROVED** [review]
This looks OK, although I'm still wondering what the plan is for the codeable concept fields. Will the ETL unify these fields so one search value can be used across all ETL'd data? Or will they prevent users from doing a search across data from different DCs?

File: src/schema/json/datatypes.json (hunk 1 above)
Comment (kreinold) [reply src/schema/json/datatypes.json:7]: Good question.  The fact is we won't for CDA MVP.  We are working with fixed baseline for now.  We will be adding lots of fields to these classes to support the CDA search needs.  Until CCDH incorporates the fields needed for CDA, we will do our best to keep up with CCDH changes.  Eventually, we hope to use their model directly.

Comment (druth314) [reply src/schema/json/datatypes.json:7]: I'm still not entirely sure how we will implement the codeable concept in BigQuery, but we'll try to get as close as we can to what CCDH has in mind

**Review by druth314 — APPROVED** [review]
I don't see any noticeable issues with the schema so far. We will have a few of these columns missing for POC but it shouldn't be an issue

**Pull request merged**